sites/frontend/components/Header/Nav/MainMenu/MainMenuColumns/MainMenuColumn/MainMenuColumnLinks/MainMenuLink/MainMenuLinkTitle.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import { styled } from '@guardian/guui';
+
+import { tablet, desktop } from '@guardian/pasteup/breakpoints';
+import { egyptian } from '@guardian/pasteup/fonts';
+import { pillars } from '@guardian/pasteup/palette';
+
+import type { ColumnType } from '../../../../../../Nav/__config__';
```

Comment:
Maybe we should try to avoid relative paths, which can be hard to follow. Do we not resolve paths from, say, `components`?

Reply:
We don't right now, but I agree relative paths are a nightmare to keep maintained when moving folder around. I can update the `webpack` config to make all import paths relative to `components`. I guess maybe we didn't do this to begin with as the vast majority of imports are sibling components or child components, but in this case  it's a 'config' we're having to go back up the tree to retrieve.

Reply:
Right on. Let's fix it when this PR is merged